src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <h1 class="visually-hidden">Search bar airbnb</h1>
+    <form action="url"
+    method="get"
+    name="big_search"
+    data-qa="big">
```

Comment:
```suggestion
    <form 
      action="url"
      method="get"
      name="big_search"
      data-qa="big"
    >
```
This is right way to style your code. Please consider it and change in **ALL** the same cases